gulp/verifier.js

```diff
@@ -74,12 +74,47 @@ module.exports.verifyTransifex = function() {
                     )
         );
         log.warn(
-        c.yellow('If you need languages other than English, you will need to create a transifex account ' + 
+        c.yellow('If you need languages other than English, you will need to create a transifex account ' +
                 'and setup the TX_USERNAME and TX_PASSWORD variables in the .env file')
         );
     }
 };
 
+module.exports.verifyEndpointStatus = function() {
+    if (isCheckDisabled('ENDPOINTS_STATUS')) {
+            log.info(c.green('USH_DISABLE_CHECKS contains ENDPOINTS_STATUS, skipping ENDPOINTS_STATUS verification process.'));
+            return;
+    }
+    const endpoints = ['tags', 'forms', 'config/features', 'config/map'];
+    const requests = endpoints.map(function(endpoint) {
+        return fetch(`${process.env.BACKEND_URL}/api/v3/${endpoint}`);
+    });
+
+    Promise.all(requests)
+    .then(function(responses) {
+        responses.forEach(function(response) {
+            log.info(c.bold(`Status-result for ${response.url}:`));
+            log.info(`The server responded with a ${response.status} code.`);
+            switch (response.status.toString()) {
```

Comment:
I think we can reuse this code together with the network-check. Just wanted it to work first.